### PR TITLE
Add new methods to ModifiableGMPE

### PR DESCRIPTION
**Pull request by g-weatherill**
The ModifiableGMPE seems to be the general solution to the implementation of scaled backbone ground motion models. This PR adds common scaling functions to the GMPE including:

1. `set_scale_median_scalar` - adjusts the median ground motions by a linear scaling factor that is independent of IMT

2. `set_scale_median_vector` - adjusts the median ground motions by a linear scaling factor that is IMT-dependent

3. `set_scale_total_sigma_scalar` - increases/decreases the total standard deviation by a linear factor that is independent of IMT

4. `set_scale_total_sigma_vector` - increases/decreases the total standard deviation by a linear factor that is IMT-dependent

5. `set_fixed_total_sigma` - Replaces output total sigma with a homoskedastic, IMT-dependent value

### CODE REVIEW

**Review by mmpagani — APPROVED** [review]
LGTM

**Comment by micheles** [comment]
Please update the changelog

File: openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py [hunk 1]
```diff
@@ -71,3 +71,106 @@ def test_get_mean_std(self):
         # Check that the total std now corresponds to the within event
         # standard deviation
         np.testing.assert_almost_equal(stds[0], estds[2])
+
+    def test_coefficients_as_dictionary(self):
+        """
+        """
```
Comment (micheles) [inline openquake/hazardlib/tests/gsim/mgmpe/modifiable_gmpe_test.py:77]: I would remove the docstrings in all tests

**Review by micheles — APPROVED** [review]

**Comment by mmpagani** [comment]
Please also update the README here https://github.com/gem/oq-engine/blob/master/openquake/qa_tests_data/classical/README.md

**Pull request merged**